### DISCUSSION TITLE
Invalid Render mode: LIT (Godot 4.0.3)

### DISCUSSION THREAD

**Discussion by Leumeister** [MenacingMecha/godot-psx-style-demo#20] [Q&A]
I installed the shaders as specified by the instructions in the Readme, but there's an error associated with `psx_base.gdshaderinc`, saying that LIT is an invalid render mode. By the looks of things, CULL and BLEND are also invalid, if their lack of color is any indication.

The Readme doesn't specify any of this, so I'm not sure how to fix. Current rendering method is set to "mobile," if that means anything.

**Comment by WJSWebster** [comment]
These are defined in the shaders before psx_base is included. See any of these:
![image](https://github.com/MenacingMecha/godot-psx-style-demo/assets/10106857/597e68f6-ef24-4b22-be59-2bc0a6a135e7)
